{{cookiecutter.project_slug}}/project/settings/production.py

```diff
@@ -60,8 +60,8 @@
 
     MIDDLEWARE = ["elasticapm.contrib.django.middleware.TracingMiddleware"] + MIDDLEWARE
 
-# Cache backed sessions for optimum performance
-SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
+# Cache (redis) sessions for optimum performance
+SESSION_ENGINE = "django.contrib.sessions.backends.cache"
```

Comment:
What do you think about putting this in `settings/base.py` as well inside the `if os.environ.get("REDIS_SERVERS"):` block?

We'd get it locally (I don't know if that's good or not).
We'd also not get this switched on if for some reason we were running without `REDIS_SERVERS`... ?

Reply:
I've moved the if statement to here.

I don't think anyone runs redis locally anyway! Usually only to test that changes like this work.

The cases where we're not running redis in production are highly likely to have a separate settings file anyway.